interactions.go

```diff
@@ -317,6 +317,8 @@ type ApplicationCommandInteractionData struct {
 	ID       string                                     `json:"id"`
 	Name     string                                     `json:"name"`
 	Resolved *ApplicationCommandInteractionDataResolved `json:"resolved"`
+	// The type of the invoked command
+	CommandType ApplicationCommandType `json:"type"`
 
```

Comment:
To be consistent with other component structs, and since the name of the field is self-explanatory, I don't think the comment is necessary here.